anno3d/annofab/constant.py

```diff
@@ -17,3 +17,9 @@ class IgnoreAdditionalDef:
 
 default_ignore_additional = IgnoreAdditionalDef("__3dpc-editor-default-ignore", "無視", "ignore", True)
 default_non_ignore_additional = IgnoreAdditionalDef("__3dpc-editor-non-default-ignore", "無視", "ignore", False)
+
+# Annofabで定義されている標準三次元エディタプラグインのID（dev / production共通）
+builtin_3d_editor_plugin_id = "bdc16348-107e-4fbc-af4a-e482bc84a60f"
+
+# Annofabで定義されている標準三次元エディタ用仕様拡張プラグインのID（dev / production共通）
+builtin_3d_extend_specs_plugin_id = "703ababa-96ac-4920-8afb-d4f2bddac7e3"
```

Comment:
APIに合わせて`extended`の方が良いと思います。
```suggestion
builtin_3d_extended_specs_plugin_id = "703ababa-96ac-4920-8afb-d4f2bddac7e3"
```

Reply:
> annofab-api-python-clientでplugin_idを定義しました。これを参照すればplugin_idの定義する必要はないでしょうか？

それはたしかにそうですね。
とはいえ、この情報を`annofab-api-python-client`が持つべきかどうかは、ちょっと怪しい気がしますが。

Reply:
変数名変更しました e319150c53a42b944613e9b3d13380196f0a14df